libs/sdk-ui-dashboard/src/model/commands/scheduledEmail.ts

```diff
@@ -0,0 +1,46 @@
+// (C) 2021 GoodData Corporation
+
+import { IScheduledMailDefinition, IFilterContextDefinition } from "@gooddata/sdk-backend-spi";
+import { IDashboardCommand } from "./base";
+
+/**
+ * Creates scheduled email.
+ *
+ * @internal
+ */
+export interface CreateScheduledEmail extends IDashboardCommand {
+    readonly type: "GDC.DASH/CMD.SCHEDULED_EMAIL.CREATE";
+    readonly payload: {
+        readonly scheduledEmail: IScheduledMailDefinition;
+        readonly filterContext?: IFilterContextDefinition;
+    };
+}
+
+/**
+ * Creates the CreateScheduledEmail command. Dispatching this command will result in the creating scheduled email on the backend.
+ *
+ * @param scheduledEmail - specify scheduled email to create.
+ * @param filterContext - specify filter context to use for the scheduled email
```

Comment:
please clarify what context will be used if none is provided

Reply:
Updated.